README.md

```diff
@@ -1 +1,61 @@
-# prometheus-exporter
\ No newline at end of file
+# Memgraph metrics exporter
+
+[Memgraph](https://github.com/memgraph/memgraph) exporter for Prometheus.
+
+The metrics currently collected are:
+
+- **TODO**
+
+## Running the exporter
+
+```shell
+$ git clone https://github.com/memgraph/prometheus-exporter.git
+$ cd prometheus-exporter
+$ python3 main.py
+```
+
+## Basic Prometheus Configuration
+
+Add Memgraph target to the Prometheus `scrape_configs`:
+
+```yaml
+scrape_configs:
+  - job_name: 'memgraph-metrics'
+  static_configs:
+    - targets: ['<<MEMGRAPH_EXPORTER_HOST>>:8000']
+```
```

Comment:
As this is already part of the `config/prometheus.yml` script, I am not sure if we need this step? 

Maybe you can say something along the lines: "In case you are running Prometheus by following our steps from below, you don't need to do anything here. But otherwise, jump to the Prometheus page to set up `scrape_configs`"